rules/use-t-well.js

```diff
@@ -3,35 +3,91 @@ const {visitIf} = require('enhance-visitors');
 const util = require('../util');
 const createAvaRule = require('../create-ava-rule');
 
-const isMethod = name => util.executionMethods.has(name);
+class MicroCorrecter {
```

Comment:
Code comment about what this does.

Reply:
Any chance you could make this a reusable npm package so we don't have to maintain the logic here? Totally optional though.